lib/region_config_data.json

```
@@ -43,8 +43,13 @@
       "globalEndpoint": true,
       "signingRegion": "cn-north-1"
     },
-    "us-gov-*/iam": "globalGovCloud",
+    "us-iso-*/iam": {
+      "endpoint": "{service}.us-iso-east-1.c2s.ic.gov",
+      "globalEndpoint": true,
+      "signingRegion": "us-east-1"
```

Should this be us-iso-east-1?
```suggestion
      "signingRegion": "us-iso-east-1"
```

---

Verified that it should be `us-iso-east-1` from Endpoints 2.0 tests https://github.com/aws/aws-sdk-js-v3/blob/7ed7101dcc4e81038b6c7f581162b959e6b33a04/codegen/sdk-codegen/aws-models/iam.json#L2270-L2291

---

Yes, thanks for catching that!